DTIPrep/DTI/DTI.pm

```diff
@@ -746,18 +888,22 @@ sub fetch_header_info {
 }
 
 
+=pod
 
+=head3 get_header_list($splitter, $fields)
 
+Get the list of arguments and values to insert into the MINC header
+(C<acquisition:*>, C<patient:*> and C<study:*>).
 
+INPUT:
+  - $splitter: splitter used to split list of fields stored in C<$fields>
```

Comment:
I would use delimiter instead of splitter

Reply:
cool!